src/ParseObject.js

```diff
@@ -309,6 +309,8 @@ class ParseObject {
     for (attr in pending[0]) {
       json[attr] = pending[0][attr].toJSON();
     }
+    json._batchIndex = this._batchIndex || 0;
```

Comment:
Will we start sending `batchIndex=0`, `batchCount=1` when it is not even a batch?

Reply:
I put this here so that you could tell that it was only 1 item being saved